src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/action/start/StartTransformRequest.kt

```diff
@@ -0,0 +1,31 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.start
+
+import org.elasticsearch.action.ActionRequestValidationException
+import org.elasticsearch.action.ValidateActions.addValidationError
+import org.elasticsearch.action.update.UpdateRequest
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+class StartTransformRequest : UpdateRequest {
```

Comment:
nit: I think you can do inline constructor 

```
class StartTransformRequest(val id: String) : UpdateRequest(id) {
```

Reply:
The above doesn't work, as it's confusing the constructor being referenced. It's trying to use the StreamInput constructor and throwing a type mismatch error on `UpdateRequest(id)`, as UpdateRequest doesn't have an ID only constructor. Currently I think it's using the no-arg constructor, then updating the ID and Index after initialization.